source/modules/.scaffold/scaffold.js

```diff
@@ -1,59 +1,60 @@
-/*!
+import EstaticoModule from '../../assets/js/helpers/module';
+
+const name = '{{name}}';
+
+/*
  * {{originalName}}
  *
  * @author
  * @copyright
  */
-import EstaticoModule from '../../assets/js/helpers/module';
-
-class {{className}} extends EstaticoModule {
+export default class {{className}} extends EstaticoModule {
+	static events = {
```

Comment:
While I certainly like the idea, could a stage 2 proposal be a bit too experimental to be used in production already? https://github.com/tc39/proposal-class-public-fields

Judging from https://github.com/babel/babel/search?q=static+property&type=Issues&utf8=%E2%9C%93, there might still be some cases where the transpiled result might not be the expected one.